### PR TITLE
Fix marathon failure

### PR DESCRIPTION
**Pull request by Bonczidai**
add sleep to wait for app startup: https://circleci.com/gh/CloudSlang/cloud-slang-content/1462 

Signed-off-by: Bonczidai Levente <levente.bonczidai@hp.com>

### CODE REVIEW

File: test/io/cloudslang/marathon/test_create_app.sl [hunk 1]
```diff
@@ -98,9 +98,14 @@ flow:
                 required: false
              - json_file
          navigate:
-           SUCCESS: list_marathon_apps
+           SUCCESS: wait_for_app_startup
            FAILURE: FAIL_TO_CREATE
 
+    - wait_for_app_startup:
```
Comment (meirwah) [inline test/io/cloudslang/marathon/test_create_app.sl]: why not use io.cloudslang.base.network.verify_app_is_up ?

Comment (Bonczidai) [inline test/io/cloudslang/marathon/test_create_app.sl]: as far as I know, the sample app we use in this example is not a webapp, it does not have a port associated with it so we cannot use `io.cloudslang.base.network.verify_app_is_up`

**Comment by meirwah** [comment]
issues in CircleCI:

02:09:28 [ERROR] There was an error running the end task execution step of: 'get_cluster_info'. Error is: Error binding output: 'number_of_nodes_in_cluster',
	Error is: Error in running script expression or variable reference, for expression: 'docker_info.split('Nodes: ')[1].split('\n')[0]
',
	Script exception is: javax.script.ScriptException: IndexError: index out of range: 1 in <script> at line number 1
02:09:28 [ERROR] Failed to run the action! Class: io.cloudslang.lang.runtime.steps.TaskSteps, method: endTask, reason: Error running: 'get_cluster_info': Error binding output: 'number_of_nodes_in_cluster',

https://circleci.com/gh/CloudSlang/cloud-slang-content/1469

**Comment by meirwah** [comment]
:+1:

**Pull request merged**